test/Y2ModuleConfig_test.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+require "yaml"
+
+Yast.import "Y2ModuleConfig"
+Yast.import "Desktop"
+Yast.import "Profile"
+
+include Yast::Logger
+
+FIXTURES_PATH = File.join(File.dirname(__FILE__), 'fixtures')
```

Comment:
I'd prefer to move those constants to the `describe Yast::Y2ModuleConfig` just to not clobber the top-level. But it's just a matter of taste.